puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_1_17_stable.yaml

```diff
@@ -0,0 +1,46 @@
+- job:
+    name: test_proxy_1_17_stable
+    project-type: matrix
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 16
+    properties:
+      - github:
+          url: https://github.com/theforeman/smart-proxy
+    scm:
+      - foreman-proxy:
+          branch: '1.17-stable'
+    triggers:
+      - scm_fifteen_minutes
+      - github
+      - schedule_failed_builds
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+          - 2.0
+          - 2.1
+          - 2.2
+          - 2.3
+          - 2.4
+      - axis:
+          type: user-defined
+          name: puppet
+          values:
+          - 3.4.0
```

Comment:
We can drop Puppet 3.4.0, 4.2.0 and 4.4.0. Add to that 4.10.9 and 5.3.3 so we test all the major versions but only their latest ones.